src/Sylius/Bundle/ApiBundle/Tests/CommandHandler/ChangeAddressOrderHandlerTest.php

```diff
@@ -0,0 +1,195 @@
+            'CocaCola',
+            'Green Avenue',
+            'US',
+            'New York',
+            '00000',
+            '123456789',
+            '999',
+            'east'
+        );
+
+        $locale = new Locale();
+        $locale->setCode('en_US');
+
+        $localeRepository->add($locale);
+
+        $currency = new Currency();
+        $currency->setCode('PLN');
+
+        $currencyRepository->add($currency);
+
+        $channel = new Channel();
+        $channel->setCode('CHANNEL');
+        $channel->setName('channel');
+        $channel->setTaxCalculationStrategy('order_items_based');
+        $channel->setDefaultLocale($locale);
+        $channel->setBaseCurrency($currency);
+
+        $channelRepository->add($channel);
+
+        $order = new Order();
+        $order->setCurrencyCode('PLN');
+        $order->setLocaleCode('en_US');
+        $order->setBillingAddress($oldBillingAddress);
+        $order->setShippingAddress($oldBillingAddress);
+        $order->setTokenValue('token');
+        $order->setChannel($channel);
+
+        $orderRepository->add($order);
+
+        $addressOrder = new AddressOrder('john@doe.com', $oldBillingAddress);
+        $addressOrder->setOrderTokenValue('token');
+
+        $addressOrderHandler($addressOrder);
+
+        $newBillingAddress = $this->createNewAddress(
+            'John',
+            'Bidd',
+            'CocaCola',
+            'Green Avenue',
+            'US',
+            'Washington',
+            '11111',
+            '100100100',
+            '111',
+            'west'
+        );
+
+        $addressOrder = new AddressOrder('john@doe.com', $newBillingAddress);
+        $addressOrder->setOrderTokenValue('token');
+
+        $addressOrderHandler($addressOrder);
+
+        /** @var AddressInterface $addressFromDatabase */
+        foreach ($addressRepository->findAll() as $addressFromDatabase) {
+            if (
+                $addressFromDatabase->getFirstName() !== $newBillingAddress->getFirstName() ||
+                $addressFromDatabase->getLastName() !== $newBillingAddress->getLastName() ||
+                $addressFromDatabase->getCompany() !== $newBillingAddress->getCompany() ||
+                $addressFromDatabase->getStreet() !== $newBillingAddress->getStreet() ||
+                $addressFromDatabase->getCity() !== $newBillingAddress->getCity() ||
+                $addressFromDatabase->getPostcode() !== $newBillingAddress->getPostcode() ||
+                $addressFromDatabase->getPhoneNumber() !== $newBillingAddress->getPhoneNumber() ||
+                $addressFromDatabase->getCountryCode() !== $newBillingAddress->getCountryCode() ||
+                $addressFromDatabase->getProvinceName() !== $newBillingAddress->getProvinceName() ||
+                $addressFromDatabase->getProvinceCode() !== $newBillingAddress->getProvinceCode()
+            ) {
+                throw new \InvalidArgumentException('The store has other address than should');
```

Comment:
```suggestion
                throw new \InvalidArgumentException('The store has other address than it should');
```
i think it looks better :D